tools/ci_credentials/README.md

```diff
@@ -0,0 +1,35 @@
+# CI Credentials
```

Comment:
+1 to no more hazing people by having them figure out these setup instructions by themselves